### PR TITLE
Add simple-nginx example

### PR DESCRIPTION
**Pull request by jijun2**
Add simple-nginx example, it is easy enough to use and understand.

### CODE REVIEW

**Comment by swagiaal** [comment]
@jijun2 from shippable:

```
kubectl-dash-f:
47: target file "nginx-controller.yaml" does not exist
102: target file "nginx-service-clusterip.yaml" does not exist

sync-examples:
open examples/simple-nginx/pod.yaml: no such file or directory

FAIL: changes needed but not made due to --verify
/root/workspace/src/k8s.io/kubernetes/examples/ is out of date. Please run hack/update-generated-docs.sh
```

File: examples/simple-nginx/README.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Running your first containers in Kubernetes
+
+Ok, you've run one of the [getting started guides](../docs/getting-started-guides/) and you have
```
Comment (thockin) [inline examples/simple-nginx/README.md]: Starting a sentence with "OK" is a bit casual for such a doc.  Maybe "By now you've run one of ..."

**Comment by thockin** [comment]
Thanks for writing docs!  This overlaps a lot with docs/user-guide/walkthrough - maybe you can make that better by folding in some of what you've done here?

**Comment by jijun2** [comment]
@erictune ,@thockin ,Writing simple-nginx is mainly for first use and understand proxy&service , many people like me is using private cluster to study and test kubernetes, maybe I should  write a good doc about simple-nginx. I think simple nginx is  very desired by newcomer!  It is valuable for newcomer use and study kubernetes.  Thanks.

**Comment by jijun2** [comment]
Example simple nginx is easy and simple for newcomer to use, this is very desired by newcomer like me !
I search a simple example for a longtime, other example is too complex for newcommer! Many
If a newcomer want to run first kubernetes app, it must be easy, simple and reliable, this is reason why I  collect and improve simple nginx example.

**Comment by thockin** [comment]
The user-guide/walkthrough is also for first-time users.

**Comment by jijun2** [comment]
@ thockin,  user-guide/walkthrough also contains redis and volumes.

**Pull request closed without merging**